source/manual/debian-packaging.html.md

```diff
@@ -10,6 +10,95 @@ review_in: 6 months
 
 This page explains how we're managing our Debian packaging.
 
+## Considerations before creating or importing a package
+
+Due to limited availability of up-to-date packages in long term support (LTS)
+distributions, requirements of software may make installation of a package
+outside of the distribution infrastructure (currently Ubuntu "Trusty" 14.04
+LTS) necessary.
+
+GDS developers may also choose to use the Debian packaging system as a means to
+distribute and maintain their locally developed software.
+
+As always when considering the addition of new software, the first question has to be:
+`is this really necessary?` Adding a package should be motivated by
+security or architecture requirements and aim to minimise unnecessary
+dependencies.
+
+Additional guidance can be found in the [GDS way](https://gds-way.cloudapps.digital/standards/tracking-dependencies.html#dependency-management-tools)
+as well as the [service manual](https://www.gov.uk/service-manual/technology/managing-software-dependencies#how-to-work-with-third-party-code).
+
+If you are satisfied that adding a local or third party package via aptly is the
+optimal solution, there are a number of ways packages may be sourced and added,
+all carrying different advantages and disadvantages as well as security and
+maintenance implications.
+
+### Creating and maintaining packages locally
```

Comment:
Might want to add something here like "e.g. through launchpad / fpm" and link to that section just to tie that a locally created package means with the tools we use.

Reply:
Similar for other sections - may also want to have links in those sections like "when to create a mirror" which links back to this header (but this might seem like overkill)